dag-pb/src/dag_pb.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package dag_pb;
+
+// An IPFS MerkleDAG Link
+message PBLink {
+  // multihash of the target object
+  bytes Hash = 1;
+  // utf string name. should be unique per object
+  string Name = 2;
+  // cumulative size of target object
+  uint64 Tsize = 3;
+}
+
+// An IPFS MerkleDAG Node
+message PBNode {
+  // refs to other objects
+  repeated PBLink Links = 2;
+  // opaque user data
+  bytes Data = 1;
+}
```

Comment:
This file itself has been updated in the spec with different comments: https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-pb.md